qutip/tests/solver/test_mesolve.py

```diff
@@ -194,14 +194,41 @@ def testME_TDDecayliouvillian(self, c_ops):
         fid = fidelitycheck(out1, out2, rho0vec)
         assert fid == pytest.approx(1., abs=me_error)
 
+    @pytest.mark.parametrize(['state_type'], [
+        pytest.param("ket", id="ket"),
+        pytest.param("dm", id="dm"),
+        pytest.param("unitary", id="unitary"),
+    ])
+    def test_mesolve_normalization(self, state_type):
+        # non-hermitean H causes state to evolve non-unitarily
+        H = qutip.Qobj([[1, -0.1j], [-0.1j, 1]])
+        H = qutip.liouvillian(H)  # ensure use of McSolve
```

Comment:
```suggestion
        H = qutip.Qobj([[1, -0.1j], [-0.1j, 1]])
        H = qutip.sprepost(H, H) # ensure use of MeSolve
```
The liouvillian is always trace preserving even when the Hamiltonian is not hermitian because we use:
`-1j * (spre(H) - spost(H))` without calling `H.dag()`...